package.json

```diff
@@ -4,13 +4,15 @@
   "description": "The service backend for the Philanthropy Data Commons",
   "main": "dist/index.js",
   "scripts": {
-    "build": "npm run cleanup && npm run build:node",
+    "build": "npm run cleanup && npm run build:node && npm run build:assets",
     "build:node": "tsc -p tsconfig.json",
+    "build:assets": "copyfiles -u 1 \"src/**/*.sql\" \"src/**/.keep\" dist",
     "cleanup": "rimraf dist",
     "lint": "eslint ./src --ext .ts",
-    "migrate": "ts-node -r dotenv/config src/scripts/migrate.ts",
+    "migrate": "npm run build && node -r dotenv/config dist/scripts/migrate.js",
     "test": "jest",
-    "start": "node ."
+    "start": "npm run build && node ./dist/index.js",
```

Comment:
Same as above. But neither of these prevent merge because we are no worse off than before, we're still better off.